jkube-kit/generator/webapp/src/test/java/org/eclipse/jkube/generator/webapp/WebAppGeneratorTest.java

```diff
@@ -126,7 +129,7 @@ void withDefaultHandler_shouldAddImageConfiguration() throws IOException {
             }})
           .extracting(BuildConfiguration::getAssembly)
           .hasFieldOrPropertyWithValue("excludeFinalOutputArtifact", true)
-          .extracting("inline.files").asList().extracting("destName")
+          .extracting("inline.files").asInstanceOf(InstanceOfAssertFactories.list(Type.class)).extracting("destName")
```

Comment:
You need to check what's the type of field in [AssemblyConfiguration](https://github.com/eclipse-jkube/jkube/blob/master/jkube-kit/common/src/main/java/org/eclipse/jkube/kit/common/AssemblyConfiguration.java#L100)'s `inline` field, then check what's type of `files` inside `inline` (`Assembly`)

https://github.com/eclipse-jkube/jkube/blob/e1cde0bd0d4ab1a659db7b1f7a884705048b27de/jkube-kit/common/src/main/java/org/eclipse/jkube/kit/common/Assembly.java#L53

You should use this type in `InstanceOfAssertFactories.list()` call